### PR TITLE
Howto reform

### PR DESCRIPTION
**Pull request by keirthana**
Some small changes from redoing the howto topics

### CODE REVIEW

File: howto/update/upgrade-appliance.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Before you upgrade the Anbox Cloud Appliance, you should make sure all packages on the machines that are part of the deployment are up-to-date. To do so, run the following commands on each machine:
+Before you upgrade the Anbox Cloud Appliance, make sure all packages on the machines that are part of the deployment, are up-to-date. To do so, run the following commands on each machine:
```
Comment (ajanon) [inline howto/update/upgrade-appliance.md:1]: Why did you add a comma there? It looks weird to me, so I'm not sure if it's intended.

**Review by ajanon — COMMENTED** [review]
Just one small question.

File: howto/update/upgrade-appliance.md (hunk 1 above)
Comment (keirthana) [reply howto/update/upgrade-appliance.md]: It was a typo, thanks!

**Review by ajanon — APPROVED** [review]
LGTM!

**Review by morphis — APPROVED** [review]
LGTM
